src/services/claim/claim.service.ts

```diff
@@ -0,0 +1,261 @@
+          amountDenorm: parseUnits(report[1][account], 18)
+        };
+      });
+
+    const availableToClaim = claims
+      .map(claim => parseFloat(claim.amount))
+      .reduce((total, amount) => total.plus(amount), bnum(0))
+      .toString();
+
+    return {
+      claims,
+      reports,
+      tokenClaimInfo,
+      availableToClaim
+    };
+  }
+
+  public async getMultiTokensCurrentRewardsEstimate(
+    account: string
+  ): Promise<{
+    data: MultiTokenCurrentRewardsEstimate[];
+    timestamp: string | null;
+  }> {
+    try {
+      const response = await axios.get<
+        MultiTokenCurrentRewardsEstimateResponse
+      >(
+        `https://api.balancer.finance/liquidity-mining/v1/liquidity-provider-multitoken/${account}`
+      );
+      if (response.data.success) {
+        const multiTokenLiquidityProviders = response.data.result[
+          'liquidity-providers'
+        ]
+          .filter(incentive => incentive.chain_id === networkId.value)
+          .map(incentive => ({
+            ...incentive,
+            token_address: getAddress(incentive.token_address)
+          }));
+
+        const multiTokenCurrentRewardsEstimate: MultiTokenCurrentRewardsEstimate[] = [];
+
+        const multiTokenLiquidityProvidersByToken = Object.entries(
+          groupBy(multiTokenLiquidityProviders, 'token_address')
+        );
+
+        for (const [
+          token,
+          liquidityProvider
+        ] of multiTokenLiquidityProvidersByToken) {
+          const rewards = liquidityProvider
+            .reduce(
+              (total, { current_estimate }) => total.plus(current_estimate),
+              bnum(0)
+            )
+            .toString();
+
+          const velocity =
+            liquidityProvider
+              .find(liquidityProvider => Number(liquidityProvider.velocity) > 0)
+              ?.velocity.toString() ?? '0';
+
+          if (Number(velocity) > 0) {
+            multiTokenCurrentRewardsEstimate.push({
+              rewards,
+              velocity,
+              token: getAddress(token)
+            });
+          }
+        }
+
+        return {
+          data: multiTokenCurrentRewardsEstimate,
+          timestamp: response.data.result.current_timestamp
+        };
+      }
+    } catch (e) {
+      console.log('[Claim] Current Rewards Estimate Error', e);
+    }
+    return {
+      data: [],
+      timestamp: null
+    };
+  }
+
+  public async multiTokenClaimRewards(
+    provider: Web3Provider,
+    account: string,
+    multiTokenPendingClaims: MultiTokenPendingClaims[]
+  ): Promise<TransactionResponse> {
+    try {
+      const [multiTokenClaims] = await Promise.all(
+        multiTokenPendingClaims.map(tokenPendingClaims =>
+          this.computeClaimProofs(tokenPendingClaims, account)
+        )
+      );
+
+      return sendTransaction(
+        provider,
+        configs[networkId.value].addresses.merkleRedeem,
+        MerkleRedeem__factory.abi,
+        'claimWeeks',
```

Comment:
should now be `claimDistributions` https://github.com/balancer-labs/balancer-v2-monorepo/blob/097740a29f8ab0e53b115b9e8aa1a73560ce54aa/pkg/distributors/contracts/MerkleOrchard.sol#L163